### PR TITLE
Update mapper formatting for DBProtect, Netsparker

### PR DESCRIPTION
**Pull request by charleshu-8**
Signed-off-by: charleshu-mitre <chu@mitre.org>

### CODE REVIEW

File: libs/hdf-converters/src/netsparker-mapper.ts [hunk 1]
```diff
@@ -201,25 +196,55 @@ export class NetsparkerMapper extends BaseConverter {
...
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      transformer: (data: Record<string, unknown>): Record<string, unknown> => {
+        const auxData = _.get(data, 'netsparker-enterprise');
+        const genData = _.get(auxData, 'generated');
+        const targetData = _.omit(_.get(auxData, 'target'), [
```
Comment (Rlin232) [inline libs/hdf-converters/src/netsparker-mapper.ts:224]: Could potentially just consolidate this to `_.get(data, 'netsparker-enterprise.generated')` and `_.get(data, 'netsparker-enterprise.target'`, but I feel like that might just be personal preference.

**Review by Rlin232 — APPROVED** [review]

**Review by Amndeep7 — CHANGES_REQUESTED** [review]
I feel very uncomfortable making changes to the mapper when we don't have the ability to test and verify what the resulting output will look like.  We can talk on Monday for what this will look like, but I think we should a) see if we know who's using it who can lend us a sample, b) scour the internet to see if there's a sample uploaded online somewhere, and/or c) reach out to the company and request the schema for their output (happy path), limited time demo of their tool so we can run it against our own systems for a day or two (medium path), a sample output (worst case still good path).

**Comment by charleshu-8** [comment]
> I feel very uncomfortable making changes to the mapper when we don't have the ability to test and verify what the resulting output will look like. We can talk on Monday for what this will look like, but I think we should a) see if we know who's using it who can lend us a sample, b) scour the internet to see if there's a sample uploaded online somewhere, and/or c) reach out to the company and request the schema for their output (happy path), limited time demo of their tool so we can run it against our own systems for a day or two (medium path), a sample output (worst case still good path).

I have a working sample on-hand for both DbProtect and Netsparker that I received from @ejaronne and have been testing my changes against those. If you want me to send them to you for review purposes, just let me know.

**Comment by charleshu-8** [comment]
HANDOFF: Blocked by Heimdall2 PR #3273 (https://github.com/mitre/heimdall2/pull/3273); otherwise, should be good

**Comment by charleshu-8** [comment]
HANDOFF: Waiting on confirmation on usage of DBP & NS data for regression testing; otherwise, is ready for merging

**Comment by charleshu-8** [comment]
Separating out the mapper updates so I can deal with them individually as the sample files come in

**Comment by charleshu-8** [comment]
Separating out the mapper updates so I can deal with them individually as the sample files come in

**Pull request closed without merging**